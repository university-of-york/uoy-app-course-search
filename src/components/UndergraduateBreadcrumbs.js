const UndergraduateBreadcrumbs = () => {
    return (
        <nav aria-label="Breadcrumb" className="c-breadcrumb">
            <div className="c-breadcrumb__items">
                <a href="https://www.york.ac.uk">
                    <i className="c-icon c-icon--home" title="Home" />
                    <span className="c-icon__sr-text">Home</span>
                </a>
                <span className="c-breadcrumb__separator">&gt;</span>
                <a href="https://www.york.ac.uk/study/">Study at York</a>
                <span className="c-breadcrumb__separator">&gt;</span>
                <a href="https://www.york.ac.uk/study/undergraduate/">Undergraduate</a>
                <span className="c-breadcrumb__separator">&gt;</span>
                <a href="https://www.york.ac.uk/study/undergraduate/courses/">Courses 2022/23</a>
            </div>
        </nav>
    );
};

export { UndergraduateBreadcrumbs };
