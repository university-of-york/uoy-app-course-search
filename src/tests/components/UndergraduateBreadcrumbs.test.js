import { render } from "@testing-library/react";
import { UndergraduateBreadcrumbs } from "../../components/UndergraduateBreadcrumbs.js";

describe("UndergraduateBreadcrumbs", () => {
    it("renders as expected", () => {
        const { asFragment } = render(<UndergraduateBreadcrumbs />);

        expect(asFragment()).toMatchSnapshot();
    });
});
