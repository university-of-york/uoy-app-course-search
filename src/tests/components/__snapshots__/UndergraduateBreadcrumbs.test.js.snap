// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UndergraduateBreadcrumbs renders as expected 1`] = `
<DocumentFragment>
  <nav
    aria-label="Breadcrumb"
    class="c-breadcrumb"
  >
    <div
      class="c-breadcrumb__items"
    >
      <a
        href="https://www.york.ac.uk"
      >
        <i
          class="c-icon c-icon--home"
          title="Home"
        />
        <span
          class="c-icon__sr-text"
        >
          Home
        </span>
      </a>
      <span
        class="c-breadcrumb__separator"
      >
        &gt;
      </span>
      <a
        href="https://www.york.ac.uk/study/"
      >
        Study at York
      </a>
      <span
        class="c-breadcrumb__separator"
      >
        &gt;
      </span>
      <a
        href="https://www.york.ac.uk/study/undergraduate/"
      >
        Undergraduate
      </a>
      <span
        class="c-breadcrumb__separator"
      >
        &gt;
      </span>
      <a
        href="https://www.york.ac.uk/study/undergraduate/courses/"
      >
        Courses 2022/23
      </a>
    </div>
  </nav>
</DocumentFragment>
`;
